core/src/test/kotlin/com/alecstrong/sql/psi/core/FixturesTest.kt

```diff
@@ -91,6 +91,7 @@ class FixturesTest(val dialect: DialectPreset, val name: String, val fixtureRoot
       DialectPreset.SQLITE_3_18 to arrayOf("src/test/fixtures", "src/test/fixtures_upsert_not_supported", "src/test/fixtures_sqlite_3_18"),
       DialectPreset.SQLITE_3_24 to arrayOf("src/test/fixtures", "src/test/fixtures_sqlite_3_24"),
       DialectPreset.SQLITE_3_25 to arrayOf("src/test/fixtures", "src/test/fixtures_sqlite_3_24", "src/test/fixtures_sqlite_3_25"),
+      DialectPreset.SQLITE_3_35 to arrayOf("src/test/fixtures", "src/test/fixtures_sqlite_3_25", "src/test/fixtures_sqlite_3_35"),
```

Comment:
Is there a reason it's missing `"src/test/fixtures_sqlite_3_24"`?

Reply:
good catch

Reply:
Out of curiosity, why does it need 24 but not 18?

Reply:
it should probably also have 18